operator-common/src/test/java/io/strimzi/operator/common/operator/resource/AbstractNonNamespacedResourceOperatorTest.java

```diff
@@ -100,7 +100,7 @@ public void testCreateWhenExistsWithChangeIsAPatch(VertxTestContext context) {
         HasMetadata hasMetadata = mock(HasMetadata.class);
         when(mockResource.get()).thenReturn(resource);
 
-        when(mockResource.withPropagationPolicy(true ? DeletionPropagation.FOREGROUND : DeletionPropagation.ORPHAN)).thenReturn(mockR);
+        when(mockResource.withPropagationPolicy(DeletionPropagation.FOREGROUND)).thenReturn(mockR);
```

Comment:
Why change this?

Reply:
Well, `true ? DeletionPropagation.FOREGROUND : DeletionPropagation.ORPHAN` is equal to 
```
if (true) {
    doSomething();`
    } else {
    doSomethingElse();
}
``` 
so it never gets to the else branch, does it?

Reply:
Good point. I didn't noticed it was just `true` and not some actual condition. Weird code :-o.